rtl/eclrtl/rtlrecord.cpp

```diff
@@ -208,25 +182,39 @@ RtlRecord::RtlRecord(const RtlFieldInfo * const *_fields, bool expandFields) : f
         {
             const RtlFieldInfo * * allocated  = new const RtlFieldInfo * [numFields+1];
             fields = allocated;
-            const RtlFieldInfo * * target = expandNestedRows(allocated, originalFields);
+            const RtlFieldInfo * * target = expandNestedRows(nullptr, allocated, originalFields);
```

Comment:
I originally had the expansion just in the lookup table, but I needed the results from getname (on the dest) to match the ones in the lookupTable (in the source). This was before I implemented the recursive solution for type_record - the translator can now work with either expanded or unexpanded RtlRecord (though it does need them to match, I suspect), so this change can be reverted if we want. The overhead is only at startup though so not sure it is that significant?